remix.init/index.js

```diff
@@ -105,71 +78,68 @@ const updatePackageJson = ({ APP_NAME, isTypeScript, packageJson }) => {
     devDependencies: removeUnusedDependencies(
       devDependencies,
       // packages that are only used for linting the repo
-      ["eslint-plugin-markdown"].concat(
-        isTypeScript ? [] : ["ts-node"]
-      )
+      ['eslint-plugin-markdown', 'eslint-plugin-prefer-let'],
     ),
-    scripts: isTypeScript
-      ? { ...scripts, typecheck, validate }
-      : { ...scripts, validate: validate.replace(" typecheck", "") },
+    scripts,
   });
 };
 
-const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
+const main = async ({ packageManager, rootDirectory }) => {
   const pm = getPackageManagerCommand(packageManager);
-  const FILE_EXTENSION = isTypeScript ? "ts" : "js";
-
-  const README_PATH = path.join(rootDirectory, "README.md");
-  const EXAMPLE_ENV_PATH = path.join(rootDirectory, ".env.example");
-  const ENV_PATH = path.join(rootDirectory, ".env");
-  const DEPLOY_WORKFLOW_PATH = path.join(
-    rootDirectory,
-    ".github",
-    "workflows",
-    "deploy.yml"
-  );
-  const CYPRESS_SUPPORT_PATH = path.join(rootDirectory, "cypress", "support");
-  const CYPRESS_COMMANDS_PATH = path.join(
+
+  const README_PATH = path.join(rootDirectory, 'README.md');
+  const FLY_TOML_PATH = path.join(rootDirectory, 'fly.toml');
```

Comment:
We will not be using fly for deployment so you can remove everything related to that. Also scan to see other tools we will not be using and remove them from here

Reply:
Could you please give this one a quick cursory glance, ive gone ahead and removed a lot of the things you originally removed as part of your stack trim PR but i might have missed a few things 🙏🏼